### DISCUSSION TITLE
Trained AI support agent with text

### DISCUSSION THREAD

**Discussion by AlexGrabovaj** [deepgram/community#964] [General help] [answered]
Hi everyone, 

does Deepgram have any feature where I can train the support agent I'm trying to build with information from documents - documentation, books, etc - so that it can serve users on that explicit domain? 

So far I've found that it serves APIs for text-to-speech, speech-to-test and intelligent processing based on pre-trained models, but nothing about training the model with data relating to one's topic.

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- The deepgram product you are using (e.g Speech to Text, Agent API)
- A request ID that triggered your error or issue.

**Comment by jpvajda** [comment, marked as answer]
@AlexGrabovaj  [Model Training is available to Enterprise Customers](https://developers.deepgram.com/docs/model#custom) only at this time, if you are interested in speaking with our Sales Team about this just let us know as we can connect you.
